File: JavaScript/1012.js
let input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8');
let lines = input.split('\n');
let valores = lines[0].split(' ');

let A = parseFloat(valores.shift());
let B = parseFloat(valores.shift());
let C = parseFloat(valores.shift());

let triangulo = (A * C) / 2;
let circulo = 3.14159 * Math.pow(C, 2);
let trapezio = (A + B) * C / 2;
let quadrado = Math.pow(B, 2);
let retangulo = A * B;

console.log(`TRIANGULO: ${triangulo.toFixed(3)}`);
console.log(`CIRCULO: ${circulo.toFixed(3)}`);
console.log(`TRAPEZIO: ${trapezio.toFixed(3)}`);
console.log(`QUADRADO: ${quadrado.toFixed(3)}`);
console.log(`RETANGULO: ${retangulo.toFixed(3)}`);

File: Python/1012.py
lines = input().split(" ")

c = float(lines.pop())
b = float(lines.pop())
a = float(lines.pop())

triangulo = a * c / 2
circulo = pow(c, 2) * 3.14159
trapezio = (a + b) * c / 2
quadrado = pow(b, 2)
retangulo = a * b

print(f"TRIANGULO: {triangulo:.3f}\nCIRCULO: {circulo:.3f}\nTRAPEZIO: {trapezio:.3f}")
print(f"QUADRADO: {quadrado:.3f}\nRETANGULO: {retangulo:.3f}")